wikidict/convert.py

```diff
@@ -251,76 +280,6 @@ def make_groups(words: Words) -> Groups:
             groups[guess_prefix(word)][word] = details
         return groups
 
-    def handle_word(self, word: str, details: Word, **kwargs: Any) -> Generator[str]:
-        name: str = kwargs["name"]
-        words: Words = kwargs["words"]
-        current_words: Words = {word: details}
-
-        # use variant definitions for a word if one variant prefix is different
-        # "suis" listed with the definitions of "être" and "suivre"
-        if details.variants:
-            found_different_prefix = False
-            for variant in details.variants:
-                if guess_prefix(variant) != name:
-                    if root_details := self.words.get(variant):
-                        found_different_prefix = True
-                        break
-            variants_words = {}
-            # if we found one variant, then list them all
-            if found_different_prefix:
-                for variant in details.variants:
-                    if root_details := self.words.get(variant):
-                        variants_words[variant] = root_details
-            if word.endswith("s"):  # crude detection of plural
```

Comment:
Since our variants "detectors" are way better now, this hack is not more useful. Given the complexity of the variants code, I prefer to delete that piece.